napari/components/_tests/test_interaction_box.py

```diff
@@ -50,3 +51,17 @@ def test_transform():
     interaction_box.transform = Affine(rotate=45)
     resulting_box = Affine(rotate=45)(resulting_box)
     np.testing.assert_equal(interaction_box._box, resulting_box)
+
+
+def test_interaction_box_changes_with_layer_transform():
+    viewer = ViewerModel()
+    image_layer = viewer.add_image(np.random.random((28, 28)))
+    image_layer.mode = 'transform'
+    initial_selection_box = np.copy(
+        viewer.overlays.interaction_box.transform.affine_matrix
+    )
+    image_layer.scale = [3, 3]
+    assert np.allclose(
+        initial_selection_box,
+        viewer.overlays.interaction_box.transform.affine_matrix,
+    )
```

Comment:
should this be `assert not np.allclose(bla?)` 🙂

Reply:
Your right. The problem is that when I used the napari_make_viewer option then it works perfect locally. When then I use the ViewerModel() the event of scale assignment is not recognized, now. The test fails now locally.
The ViewerModel have no EventModel in the test environment? @alisterburt @tlambert03